src/app/shared/constants/flags/quest-state.ts

```diff
@@ -0,0 +1,11 @@
+import { Flag } from '../../types/general';
+
+export const QUEST_STATE: Flag[] = [
+  { bit: 0, name: 'NONE (not taken)', },
+  { bit: 1, name: 'COMPLETE', },
+  { bit: 2, name: 'UNAVAILABLE (unused)', },
+  { bit: 3, name: 'INCOMPLETE (in progress)', },
+  { bit: 4, name: 'AVAILABLE (unused)', },
```

Comment:
Why are they marked as unused?

Reply:
@Kitzunu

because they seem to be unused in the core:

![image](https://user-images.githubusercontent.com/75517/82704857-ceee0200-9c76-11ea-862c-148fe6f76da3.png)

Reply:
Actually this condition doesnt even make sense because we have 
CONDITION_QUESTREWARDED
CONDITION_QUESTTAKEN
CONDITION_QUEST_NONE
CONDITION_QUEST_COMPLETE

Only one that doesn't exist is Failed
Perhaps this condition is just so we can have less queries if we want to make conditions with all of them. But then, why do they exist when we can just use this?
weird